doc/api/fs.md

```diff
@@ -746,18 +746,11 @@ changes:
 
 Tests a user's permissions for the file or directory specified by `path`.
 The `mode` argument is an optional integer that specifies the accessibility
-checks to be performed. The following constants define the possible values of
+checks to be performed. Check
+[`File Access Contants`](#fs_file_access_constants) for possible values of
```

Comment:
Contants -> Constants

Reply:
Also, `` `File Access Contants` `` -> just `File Access Contants` (as this is not a code)?